.github/workflows/send-tag-event.yml

```diff
@@ -0,0 +1,24 @@
+name: Send Tag Event
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build-and-push:
```

Comment:
I think nothing is being built and pushed here.

Reply:
oops, copy paste issue, fixed 👍